src/emulator/memory_manager.hpp

```diff
@@ -133,6 +144,15 @@ class memory_manager
     virtual void apply_memory_protection(uint64_t address, size_t size, memory_permission permissions) = 0;
 
   protected:
+#if MOMO_REFLECTION_LEVEL > 0
+    std::atomic<std::uint64_t> memory_layout_state_ver_{0};
+
+    void inc_memory_layout_state_ver()
```

Comment:
I'm not a fan of abreviations.
i think something like `increase_memory_layout_state` might be better